avail/src/main/kotlin/avail/anvil/shortcuts/AvailDebuggerShortcuts.kt

```diff
@@ -0,0 +1,104 @@
+/*
+ * WorkbenchShortcuts.kt
```

Comment:
Something seems to have happened with IntelliJ, like some kind of regression. Cloning classes using their copy doesn't change the file name in the copyright header and I think it used to do that.

Also copying and pasting portions of comments seems to be busted too when in a doc comment.

Reply:
Yeah, at least a year or two ago, I noticed that _sometimes_ when I would select and start to overwrite the file name, it would show a mangled mix of old and new name.  Like it had changed the underlying text but not updated the screen yet.  Anyhow, have a look in `doc/scripts/Code maintenance scripts.md` for a pasteable bash command that tells you which files don't have the right names inside them.